### ISSUE TITLE
Null / Missing Values in VBET Evidence CSV

### ISSUE THREAD

**Issue by shelbysawyer** [Riverscapes/riverscapes-tools#357]
We've noticed from null or missing values in the csv, and I'm curious about what's causing that. It seems to be occurring for every line or evidence apart from slope.

**Comment by joewheaton** [comment]
@shelbysawyer can you please take a little screen shot or insert a table here showing @KellyMWhitehead exactly what you're talking about and for what fields from the CSV.

**Comment by shelbysawyer** [comment]
@KellyMWhitehead Specifically, I noticed that the HAND, ChannelDist, and TWI fields are missing for some of the points. 

The pattern that I have noticed is that when a feature is missing information from these fields, it is either missing both HAND and TWI _or_ all three fields. I don't believe there are any that are only missing one field.

[Here](https://youtu.be/UYSMmZivBUc) is a video showing the spreadsheet and missing values.
